### PR TITLE
fix: explicitly initialize `Std` in `lean_initialize`

### PR DESCRIPTION
**Pull request by Kha**
Fixes the stage 2 build, which runs with `prefer_native=true`

### CODE REVIEW

File: src/Std/Data/DHashMap.lean [hunk 1]
```diff
@@ -5,5 +5,6 @@ Authors: Markus Himmel
 -/
 prelude
 import Std.Data.DHashMap.Basic
+import Std.Data.DHashMap.RawLemmas
```
Comment (Kha) [inline src/Std/Data/DHashMap.lean:8]: @TwoFX Just checking, was orphaning the file intentional?

Comment (TwoFX) [reply src/Std/Data/DHashMap.lean:8]: Yes, the idea was that in the "normal" use case where you just want to use `HashMap` and not `HashMap.Raw` you can just `import Std.Data.HashMap` and import only the files that are needed for `HashMap`.

In fact, the same thing should be true for all three of `DHashMap`, `HashMap` and `HashSet`, so I'm surprised that just changing this for `DHashMap` made any difference.

Comment (TwoFX) [reply src/Std/Data/DHashMap.lean:8]: Ah, maybe it's because `DHashMap.RawLemmas` has macros in it, unlike `HashMap.RawLemmas` and `HashSet.RawLemmas`?

Comment (Kha) [reply src/Std/Data/DHashMap.lean:8]: Yes, exactly. It's a limitation of the bootstrapping setup, if you will. But note that a Lake build would also skip orphan files.

Perhaps the import should be moved to Std.Data instead then? Unlikely people will import that and then complain that it contains too much stuff.

Comment (TwoFX) [reply src/Std/Data/DHashMap.lean:8]: Sounds good to me, I'll open a PR.

**Pull request merged**